src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+
+        StringBuilder binaryString = new StringBuilder();
+        while (value > 0) {
+            binaryString.insert(0, ((value % 2) == 0 ? "0" : "1"));
```

Comment:
try to remove ternary operator on this line and see what happens, it's not needed here, you can pass value from % 2 directly

Reply:
thanks fixed